tcms/bugs/tests/test_admin.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=invalid-name
+from django.urls import reverse
+from tcms.tests import LoggedInTestCase
+from tcms.utils.permissions import initiate_user_with_default_setups
+from tcms.tests.factories import TestRunFactory
+
+
+class TestBugAdmin(LoggedInTestCase):
+
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+        initiate_user_with_default_setups(cls.tester)
+        cls.test_run = TestRunFactory()
+
+    def test_regular_user_can_add_bug(self):
```

Comment:
Method naming doesn't match reality:
- any user can add bugs, regardless of whether or not they are superuser
- the method doesn't verify that you can actually add a new bug record to the DB, it verifies that a particular redirect is happening
- the redirect that is happening, and which is being verified is to another page, which in turn allows adding of bugs via more detailed form. And we don't care about that last part.

Change the method name so that it is clear what the actual behavior is intended to be.

Reply:
done